[1.13] Fix missing image names on list

saschagrunert opened the pull request:
Backport of #2455

rhatdan commented:
LGTM
/approve

haircommander commented:
you're getting lint errors:
` make lint
checking lint
server/image_list.go:42:1:warning: exported function ConvertImage should have comment or be unexported (golint)
Makefile:74: recipe for target 'lint' failed
make: *** [lint] Error 1
The command "make lint" exited with 2.`

haircommander commented:
otherwise LGTM

saschagrunert commented:
> you're getting lint errors:
> ` make lint checking lint server/image_list.go:42:1:warning: exported function ConvertImage should have comment or be unexported (golint) Makefile:74: recipe for target 'lint' failed make: *** [lint] Error 1 The command "make lint" exited with 2.`

Hm, I added a comment so this should be now fixed here.

saschagrunert commented:
CircleCI won't be happy since there exists no build config within that branch.

mtrmac commented on server/image_list.go:
```diff
@@ -30,22 +31,45 @@ func (s *Server) ListImages(ctx context.Context, req *pb.ListImagesRequest) (res
+	repoDigests := []string{"<none>@<none>"}
+	if len(from.RepoDigests) > 0 {
+		repoDigests = from.RepoDigests
+	}
```
I can’t see how this makes sense: https://github.com/cri-o/cri-o/pull/2455#discussion_r291677114 .

haircommander commented:
/test e2e_rhel

saschagrunert commented:
/test e2e_rhel

haircommander commented:
/lgtm

haircommander commented:
manually merging because there is no build config...
